README.rst

```diff
@@ -21,11 +21,17 @@ following entities:
 
 Installation
 ------------
+
+``pyiges`` is offered in a "full" variant including the conversion features
+and a "pure" parsing module variant.
+The pure variant has no conversion features, no dependencies to ``pyvista,geomdl``,
+and can be installed by removing the ``[full]`` specificator from the following commands.
```

Comment:
As decided in #28 to get the full package, install commands have to be updated. I gave the focus on the full install in the docs